.github/workflows/e2e.yaml

```diff
@@ -44,6 +44,8 @@ jobs:
           ./mvnw --batch-mode -f test/e2e/pom.xml -pl e2e-base clean install
       - name: Single Node Tests(JDK8)
         run: export E2E_VERSION=jdk8-1.5 && bash -x test/e2e/run.sh e2e-single-service
+      - name: Single Http API with Ningx Lua Tests(JDK8)
```

Comment:
And `Single` is not clear in the expression. Please rename this to a more clear name.